pyproject.toml

```diff
@@ -28,12 +28,12 @@ dependencies = [
   "coloredlogs",
   "dacite",
   "orjson",
-  "home-assistant-chip-clusters==2023.12.0"
+  "home-assistant-chip-clusters==2024.1.0"
```

Comment:
The wheels bump wasn't mentioned in the PR description.

Reply:
Right, I've missed mentioning that :cry: 

Added it to the PR description!